### PR TITLE
Fix SIM tests failures

### PR DESCRIPTION
**Pull request by nazarhussain**
**Motivation**

Make the sim tests more stable.

**Description**

Fix the sim tests. failures. 

**Steps to test or reproduce**

Run sim tests.

### CODE REVIEW

File: packages/cli/test/utils/simulation/cl_clients/lodestar.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ export const generateLodestarBeaconNode: CLClientGenerator<CLClient.Lodestar> =
     "network.allowPublishToZeroPeers": false,
     discv5: true,
     "network.connectToDiscv5Bootnodes": true,
+    "network.rateLimitMultiplier": 0,
```
Comment (nazarhussain) [inline packages/cli/test/utils/simulation/cl_clients/lodestar.ts:55]: It turned out that with recent changes if there is rate limit error, the peer get disconnected. Had to disable the limit for sim tests.

File: packages/cli/test/sim/multi_fork.test.ts [hunk 2]
```diff
@@ -153,6 +153,9 @@ const headForUnknownBlockSync = await env.nodes[0].cl.api.beacon.getBlockV2("hea
 ApiError.assert(headForUnknownBlockSync);
 await connectNewNode(unknownBlockSync, env.nodes);
 
+// Wait for EL node to start and sync
+await sleep(5000);
```
Comment (nazarhussain) [inline packages/cli/test/sim/multi_fork.test.ts:157]: The `uknownBlock` sync does not trigger to completion if the `EL` node is still syncing. So why had to add some wait time so EL node get sync first then we publish an uknown block.

**Review by twoeths — APPROVED** [review]

**Comment by twoeths** [comment]
Thanks @nazarhussain !

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.6.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.6.0) :tada:

**Pull request merged**